_posts/2018-11-24-why-would-a-java-engineer-love-fronted-development.md

```diff
@@ -190,15 +190,15 @@ from CLI.
 **Fast updates with minimal breaking changes**
 
 In the Java community, we are used to the fact that a code written 10 years ago
-will work on a new JVM. There are some exceptions like transition from Java 8 
-to Java 9, but generally, it's true. On the other hand, in frontend tendencies 
-are different: you take a break for 6 months and boom, everything is changed,
+will work on a new JVM. There are some exceptions like when transitioning from Java 8 
+to Java 9, but generally, it's true. On the other hand, in frontend things tend to be 
+different: you take a break for 6 months and boom, everything has changed and
 everything is new for you.
 
-Angular is somewhere in in-between. They release often, they introduce breaking
-changes but at the same time, they make transitions soft.
+Angular is somewhere in between. They release often, they introduce breaking
+changes but at the same time, they make the transitions soft.
 
-For instance, when we migrated from Angular 3 to Angular 4 we literally changed 
+For instance, when we migrated from Angular 3 to Angular 4 they literally changed 
```

Comment:
Here using "they" changes the meaning of the phrase. It is "*we*" who "*changed a few lines of code*", not "*they*".

Reply:
Ok, then "we" is correct. Sorry.